modules/building/partials/konflux-prefetch-hermeto-note.adoc

```diff
@@ -0,0 +1 @@
+IMPORTANT: Note that while there are some differences between using `hermeto` as a command-line tool, and running it in a pipeline (i.e. in {ProductName}), the most important usage pattern is the same, namely: **Always** use the JSON-formatted input when running in a pipeline environment, paying special attention to the extra options allowed by JSON input.
```

Comment:
nitpick: Looking at the rendered version of Konflux docs, having always 2 IMPORTANT notes next to each other may be a bit too much optically, so if we could put this note right after the corresponding walkthrough note and somehow make it part of the same important note would be IMO better, but I will not block on this, up to you how much more time you want to invest into this.

Reply:
Yeah, I don't love it, but I can't figure out how to force ASCIIDoc to *allow* me to have multiple paras as a single "IMPORTANT" block

Reply:
FIGURED IT OUT :muscle:

Fixed